Version 2.2.4-SNAPSHOT

tryptichon opened the pull request:
* Throw exception on detecting wrong token instead of propagating the error to the data
listener.

tryptichon committed:
Version 2.2.4-SNAPSHOT

* Throw exception on detecting wrong token instead of propagating the error to the data
listener.

tryptichon committed:
Documentation

mglusiuk commented on java/hiro-client/src/main/java/co/arago/hiro/client/rest/DefaultWebSocketClient.java:
```diff
@@ -226,7 +227,13 @@ public void onTextFrame(String payload, boolean finalFragment, int rsv) {
+                        doReconnect = false;
+                        // process(dataListener, payload);
+                        Object message = error.get("message");
+                        onError(new HiroException(
```
this is rather reserved for ws protocol errors and not graph errors
so far graph errors was always propagated as special message over data

tryptichon replied:
So maybe propagate them to the dataListener and logListener? Or shall we just reject this pull request and leave it the way it was?

mglusiuk replied:
pls do not modify error propagation.
other changes might be applied

tryptichon replied:
Ok. I rolled back to the old way but kept the renaming of the flag.

tryptichon committed:
Propagate error revoked

Propagate error to dataListener and do not throw an error. Keep error
propagation the way it was.
